docs/src/operations/best-practices/general.md

```diff
@@ -36,7 +36,7 @@ This is a monthly call that is hosted by the Solana Foundation.
 
 ### Community Led Validator Call
 
-This is also a monthly call which is hosted by the Solana validator community itself.  
```

Comment:
@kskalski that's strange, as this PR removes that whitespace here. It also passed that same sanity check on the last commit before merge.

https://buildkite.com/anza/agave/builds/28847/steps/canvas?jid=0198c7f1-29f6-4001-ab7a-f9966ff4aaf3